scripts/verify_tests_on_master.sh

```diff
@@ -29,7 +29,7 @@ flutter/bin/flutter doctor -v
 export PATH="$PATH":`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin
 
 cd flutter/dev/customer_testing
-pub get
+dart pub get
```

Comment:
Can you make the same change in verify_tests_on_master.bat?

Reply:
Done!